apps/client/components/Profile/edit/UserInfo/ProfileImage/index.tsx

```diff
@@ -31,7 +31,7 @@ const ProfileImage = () => {
         `}
       >
         <Image
-          src={edit.profile || userInfo?.profile || avatar}
+          src={edit.profile || userInfo?.profile || grayAvatar}
```

Comment:
```suggestion
          src={edit.profile ?? userInfo?.profile ?? grayAvatar}
```

해당 연산자를 사용하지 않은 이유가 있으신가용?